CMakeLists.txt

```diff
@@ -213,14 +213,14 @@ if (ENABLE_SWIG_JNI_BINDINGS OR ENABLE_SWIG_PYTHON_BINDINGS)
             SOURCES ${SwigSourcesGlob}
     )
     set_target_properties (
-            _pyosrm PROPERTIES
+            pyosrm PROPERTIES
             SWIG_USE_TARGET_INCLUDE_DIRECTORIES TRUE
-            OUTPUT_NAME _osrm
```

Comment:
_osrm will lead to __osrm.so which will cause trouble when python try to import as __osrm.